client/my-sites/design-preview/index.js

```diff
@@ -190,11 +190,13 @@ const DesignPreview = React.createClass( {
 			return null;
 		}
 
+		const previewUrl = this.getPreviewUrl( this.props.selectedSite );
+
 		return (
 			<WebPreview
 				className={ this.props.className }
-				previewUrl={ useEndpoint ? null : this.getPreviewUrl( this.props.selectedSite ) }
-				externalUrl={ this.props.selectedSite ? this.props.selectedSite.URL : null }
+				previewUrl={ useEndpoint ? null : previewUrl }
+				externalUrl={ previewUrl }
```

Comment:
`externalUrl` should be supplied an URL stripped of all iframe preview query args. At the moment I'm getting `https://ehgdesktopscreen.wordpress.com/about?iframe=true&theme_preview=true&frame-nonce=xxxx`. I *think* the only allowed query arg should be `p`, for previewing not yet published posts.